Tests/test_phenotype.py

```diff
@@ -3,9 +3,11 @@
 # This code is part of the Biopython distribution and governed by its
 # license.  Please see the LICENSE file that should have been included
 # as part of this package.
+"""Tests for the Bio.phenotype module."""
 
 try:
     import numpy
+    del numpy
 except ImportError:
     from Bio import MissingExternalDependencyError
     raise MissingExternalDependencyError(
```

Comment:
Again not direclty related to this PR, but I would sort the import stuff like this (PEP-8):

```python
import json
import unittest
import warnings

try:
    import numpy
    del numpy
except ImportError:
    from Bio import MissingExternalDependencyError
    raise MissingExternalDependencyError(
        "Install NumPy if you want to use Bio.phenotype.")

from Bio._py3k import StringIO
from Bio import BiopythonExperimentalWarning

with warnings.catch_warnings():
    warnings.simplefilter('ignore', BiopythonExperimentalWarning)
    from Bio import phenotype
```

Reply:
You mean first standard library imports, then own library imports (here Bio, BioSQL), finally any local imports (e.g. the BioSQL tests do that). I agree, we should work towards that.